core/project_analyzer.py

```
@@ -0,0 +1,259 @@
+import os
+import re
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)